test/index.ts

```diff
@@ -75,7 +82,9 @@ describe('paginator', () => {
 
     it('should accept an array or string method names', () => {
       const originalMethod = FakeClass.prototype.methodToExtend;
-      FakeClass.prototype.anotherMethodToExtend = () => {};
+      FakeClass.prototype.anotherMethodToExtend = () => {
+        // do nothing
```

Comment:
@bcoe @alexander-fenster I think this rule is annoying and we should get rid of it.  Thoughts?